nightfall-deployer/contracts/ChallengesUtil.sol

```diff
@@ -205,7 +85,10 @@ library ChallengesUtil {
     ) public pure {
         require(uint256(transaction.ercAddress) <= MAX20, 'ERC address out of range');
         require(
-            uint256(transaction.recipientAddress) <= MAX20,
+            (transaction.transactionType != Structures.TransactionTypes.TRANSFER &&
+                uint256(transaction.recipientAddress) <= MAX20) ||
+                (transaction.transactionType == Structures.TransactionTypes.TRANSFER &&
+                    uint256(transaction.recipientAddress) <= MAX31),
```

Comment:
I don't think this will be true, recipient address in transfer's can be up to the field size which is > MAX31.